### DISCUSSION TITLE
PET and UTCI calculation

### DISCUSSION THREAD

**Discussion by SimoneBeltramino** [UMEP-dev/UMEP#406] [Q&A]
Hello, I would like to know how SOWEIG calculates PET and UTCI indexes. 
What I need are the equations used by the tool. 

Thank you so much in advance. 
Simone

**Comment by biglimp** [comment]
Here you have info on UTCI: http://www.utci.org/
Regarding PET, I would take a look at: https://link.springer.com/article/10.1007/s004840050118

**Reply by SimoneBeltramino** [reply]
Thank you so much!
I've been meaning to ask you how you consider weather data? It seems to me that you SOLWEIG use the input data without modeling it on the grid (like Envi-met does for example). Is that right?

**Reply by biglimp** [reply]
As SOLWEIG is a radiation model and not a full CFD model as ENVI-met, is use the meteorological data as forcing data to estimate the radiation which is used to calculate mean radiant temperature.

**Reply by SimoneBeltramino** [reply]
One last thing, I can't find the landcoverclasses_v2016a.txt on my pc. Do you know if is available somewhere?

**Reply by biglimp** [reply]
If you are using windows and hae installed UMEP the search path should look something like this: C:\Users\ **yourusername** \AppData\Roaming\QGIS\QGIS3\profiles\default\python\plugins\UMEP\SOLWEIG

**Reply by SimoneBeltramino** [reply]
Thank you, I managed to solve everything! I take advantage of the fact that you are very helpful and kind to ask you three more things. 
1) Tmrt is calculated at what height? At ground level?
2) Modifying the parameters inside the text file landcoverclasses_v2016a.txt is it possible to make simulations with different values of emissivity and albedo?
3) Is there a limit to the points of interest that can be evaluated? I tried with just under a dozen points and it gave me this error "POI attribute with unique values must be selected".

Thank you very much,
Simone

**Reply by biglimp** [reply]
1. 1.1 meter above ground
2. yes
3. It should be possible to have as many POIs as you want but I have not tried with more than 10.

**Reply by elidamd** [reply]
Following this track, this paper (DOI 10.1007/s00704-010-0382-8) mentions future developments for Solweig 3.0 with the inclusion of user defined properties such as albedo, and emissivity, in a grid. 
Will this parametrization be implemented?
Thanks!

**Comment by biglimp** [comment]
Yes, in http://dx.doi.org/10.1007/s00484-016-1135-x.

**Reply by elidamd** [reply]
Thanks! I will test changing parameters in the text file. 
Would it be possible to increase the range of surface materials, if both text file and land cover raster correspond? 
Or the model is limited to the 7 preset?

**Reply by biglimp** [reply]
Yes, you should be able to have as many as you like.
